spec/bot_spec.rb

```diff
@@ -206,6 +206,18 @@
     subject(:handle_pull_request) { bot.handle_pull_request(action: "opened", title: title, pr_number: pr_number) }
 
     context "when linked issue exists" do
+      it "fixes the PR title if it is based on the branch name" do
```

Comment:
This test is really hard to read because the important thing here is the title but we are setting the value very far aways and not in an explicit way. 

This seem to be a case of  [Mystery guest anti-pattern](https://robots.thoughtbot.com/mystery-guest)

Can we make this more explicit?

Reply:
sure